test/functional/comment_controller_test.rb

```diff
@@ -95,9 +95,7 @@ def teardown
     UserSession.create(users(:bob))
     comment = comments(:first)
     new_comment_body = 'New body text'
-    post :update,
-         id: comment.id,
-         body: new_comment_body
+    post :update, params: { id: comment.id, body: new_comment_body }
     comment.reload
```

Comment:
`comment.reload` is this necessary here? can you try removing these too?

Reply:
Yeah I will remove that. Thanks!!

Reply:
Yeah I will remove that. Thanks!!

Reply:
Yeah I will remove that. Thanks!!

Reply:
also please make all tests use the new way of passing params around like this `post :update, params: { id: comment.id, body: new_comment_body }` i see you changed some tests manually so they pass, some of these problems are because you're still using the old way of passing around params. please change over to use the new style before going on. this is very important. @Souravirus

Reply:
Ok I will change all the tests which are passing parameters through requests to the new way of passing parameters.